app/sagas/deepLinking.js

```diff
@@ -65,21 +57,9 @@ const navigate = function* navigate({ params }) {
 				};
 
 				const isMasterDetail = yield select(state => state.app.isMasterDetail);
-				const subscribedRoom = yield select(state => state.room.subscribedRoom);
 				const jumpToMessageId = params.messageId;
 
-				if (subscribedRoom === room.rid) {
-					if (jumpToThreadId) {
-						// With this conditional when there is a jumpToThreadId we can avoid the thread open again
-						// above other thread and the room could call again the thread
-						popToRoot({ isMasterDetail });
-					}
-					yield goRoom({ item, isMasterDetail, jumpToMessageId, jumpToThreadId });
-				} else {
-					popToRoot({ isMasterDetail });
-					yield goRoom({ item, isMasterDetail, jumpToMessageId, jumpToThreadId });
-				}
-
+				yield goRoom({ item, isMasterDetail, jumpToMessageId, jumpToThreadId, popToRoot: true });
```

Comment:
I used these links to test properly the deeplinking:
    1 - rocketchat://room?host=open.rocket.chat&rid=GENERAL&messageId=LbEG77mxpL6QDKoMC
    2 - https://go.rocket.chat/room?host=open.rocket.chat&path=channel/general/thread/meRK2nfjR99MjLn55